src/components/form/templates/types.ts

```diff
@@ -66,3 +66,7 @@ export interface ArrayFieldItem {
     onReorderClick: (index: number, newIndex: number) => (event: any) => void;
     readonly: boolean;
 }
+
+export interface Runnable {
+    run: (event: any) => void;
```

Comment:
Any particular reason why actions in general wouldn't benefit from a `run` property?

Reply:
I thought about it, but it didn't seem like the right solution. There is only one listener for the action event here, but there might be several listeners in other cases and they might want to do different things. Here I added the function to the action object mostly because it seemed like the easiest solution, instead of mapping the action id to a different function inside the handler. 

I hope that made sense :smile:

Reply:
Yeah, adding the function rather than mapping the id was really nice! I did the mapping thing when I temporarily modified the example for collapsible-section in order to try it, and mapping felt a lot clunkier.